packages/manager/src/factories/objectStorage.ts

```diff
@@ -77,3 +82,11 @@ export const makeObjectsPage = (
 });
 
 export const staticObjects = objectStorageObjectFactory.buildList(250);
+
+export const getObjectStorageEndpointsFactory = Factory.Sync.makeFactory<ObjectStorageEndpoint>(
```

Comment:
Were you thinking it adds clarity to include `get` here? Our factories for get requests don't generally include the verb.

```suggestion
export const objectStorageEndpointsFactory = Factory.Sync.makeFactory<ObjectStorageEndpoint>(
```

Reply:
Typically I like to use the verbs for POST/PUT, not necessarily for GET - Good catch